lib/matplotlib/backends/backend_svg.py

```diff
@@ -68,6 +68,12 @@
 # OF THIS SOFTWARE.
 # --------------------------------------------------------------------
 
+def _sorted_iteritems(a):
+    return ((k,v) for k,v in sorted(six.iteritems(a)))
```

Comment:
This is sorting on the `(k, v)` tuple, not just the key.

It might be better to do `return ((k, a[k]) for k in sorted(a))`